apix-integrationtests/alfresco/src/main/java/eu/xenit/apix/tests/metadata/NodeServiceTestJavaApi.java

```diff
@@ -239,7 +232,7 @@ public void TestSetAssociations() {
             assertEquals(testNodeRef.toString(), assoc.getSourceRef().toString());
             assertEquals(testNodeRef2.toString(), assoc.getTargetRef().toString());
 
```

Comment:
Changed the ASSOC_TYPE here because the ASSOC_AVATAR throws errors due to the files not being a Person as target/source!